### ISSUE TITLE
8416 - 508 Audit - Defect 3 - High: Focus not applied in order

### ISSUE THREAD

**Issue by meaghan-sullivan** [department-of-veterans-affairs/va.gov-team#130359]
Defect 3 | High | Created:
WCAG 2.4.3 Focus Order | If a Web page can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability.

On the Submit Medical Expenses To Support A Pension Or DIC Claim | Veterans Affairs screen (You are on Step 1 of 4: Your information), elements do not receive focus in an order that preserves meaning and operability—the label, Country code, receives focus separately from the combo box. (New: 01/16/2026)

**Comment by maggiewachs** [comment]
I'm unable to reproduce incorrect focus order in any of the screens under "Your information". Can the original reporter provide steps to reproduce?
